### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by dthaler**
Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

File: Stack/platforms/linux/opcua_p_interface.h [hunk 1]
```diff
@@ -343,7 +343,7 @@ struct S_OpcUa_Port_CallTable
     /** @brief Get an unique id for the calling system thread.
      *  @ingroup opcua_platformlayer_interface
      */
-    unsigned long       (OPCUA_DLLCALL* ThreadGetCurrentId)       ( void);
+    OpcUa_UInt32        (OPCUA_DLLCALL* ThreadGetCurrentId)       ( void);
```
Comment (bernd-edlinger) [inline Stack/platforms/linux/opcua_p_interface.h]: please revert this change

File: Stack/proxystub/clientproxy/opcua_channel.c [hunk 2]
```diff
@@ -487,7 +487,7 @@ OpcUa_StatusCode OpcUa_Channel_InvokeService(   OpcUa_Channel           a_pChann
     OpcUa_UInt32            uTimeout            = OPCUA_INFINITE;
     OpcUa_MessageContext    cContext;
 
-OpcUa_InitializeStatus(OpcUa_Module_Channel, "InvokeService");
+OpcUa_InitializeStatus(OpcUa_Module_Channel, "OpcUa_Channel_InvokeService");
```
Comment (bernd-edlinger) [inline Stack/proxystub/clientproxy/opcua_channel.c]: please revert this chage, it is more consistent to leave the prefix away.

File: Stack/proxystub/clientproxy/opcua_channel.c [hunk 3]
```diff
@@ -939,7 +939,7 @@ OpcUa_StatusCode OpcUa_Channel_BeginConnect(OpcUa_Channel
     OpcUa_Boolean                   bLocked                 = OpcUa_False;
 #endif /* OPCUA_USE_SYNCHRONISATION */
 
-OpcUa_InitializeStatus(OpcUa_Module_Channel, "BeginConnect");
+OpcUa_InitializeStatus(OpcUa_Module_Channel, "OpcUa_Channel_BeginConnect");
```
Comment (bernd-edlinger) [inline Stack/proxystub/clientproxy/opcua_channel.c]: please leave the prefix away.

File: Stack/platforms/linux/opcua_p_interface.h (hunk 1 above)
Comment (dthaler) [reply Stack/platforms/linux/opcua_p_interface.h]: Can you say why?

File: Stack/proxystub/clientproxy/opcua_channel.c (hunk 2 above)
Comment (dthaler) [reply Stack/proxystub/clientproxy/opcua_channel.c]: Can you say why?  it is more consistent with other places to have the full function name.  If that's not desired, then a lot of other places need to change to remove the prefix.

File: Stack/proxystub/clientproxy/opcua_channel.c (hunk 3 above)
Comment (dthaler) [reply Stack/proxystub/clientproxy/opcua_channel.c]: Same question.

File: Stack/proxystub/clientproxy/opcua_channel.c (hunk 2 above)
Comment (bernd-edlinger) [reply Stack/proxystub/clientproxy/opcua_channel.c]: "OpcUa_Module_Channel" is already in the trace message, so the "OpcUa_Channel_" prefix
on InvokeService is redundant.

File: Stack/platforms/linux/opcua_p_interface.h (hunk 1 above)
Comment (bernd-edlinger) [reply Stack/platforms/linux/opcua_p_interface.h]: It won't work with 64bit linux.

File: Stack/proxystub/clientproxy/opcua_channel.c (hunk 2 above)
Comment (dthaler) [reply Stack/proxystub/clientproxy/opcua_channel.c]: Thanks for the explanations

**Comment by dthaler** [comment]
Made changes as requested
